impl/src/main/java/uk/ac/ox/oucs/vle/proxy/SakaiProxyImpl.java

```diff
@@ -391,11 +391,11 @@ public String getConfirmUrl(String signupId, String placementId) {
 		if (null == signupId) {
 			return getUrl("/static/pending.jsp", placementId);
 		}
-		return getUrl("/static/pending.jsp#"+ signupId, placementId);
+		return getUrl("/static/pending.jsp#" + signupId, placementId);
 	}
```

Comment:
Apart from the de-gibboning text changes, did you mean to put in spaces around the place like here?